src/PKSim.Core/Services/PKAnalysesTask.cs

```diff
@@ -299,6 +299,7 @@ public IReadOnlyList<PopulationPKAnalysis> AggregatePKAnalysis(Simulation simula
       {
          var pkParametersList = pkParameters.ToList();
          var matrix = new FloatMatrix();
+         //WHY THE DISTINCT HERE. THis array is then used later in buildPopulationPKAnalysis by Index. 
          var names = pkParametersList.Select(x => x.Name).Distinct().ToList();
```

Comment:
This is very weird here. In the function below (buildPopulationPKAnalysis), we are using the index of names and values where one is not created based of the other. Very weird